<ui version="4.0" >
 <class>ActivationFunction</class>
 <widget class="QWidget" name="ActivationFunction" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QRadioButton" name="globalButton" >
       <property name="text" >
        <string>Global</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="perLayerButton" >
       <property name="text" >
        <string>Per layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="individualButton" >
       <property name="text" >
        <string>Individual</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1" >
    <widget class="QComboBox" name="activationFunctionCombo" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="selectedLayerCombo" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Function</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="selectedNeuronCombo" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="layerLabel" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" >
      <string>Layer</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="settingsLabel" >
     <property name="text" >
      <string>Settings mode:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="neuronLabel" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Neuron</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Steepness</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1" >
    <widget class="QDoubleSpinBox" name="steepnessSpin" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>238</red>
           <green>238</green>
           <blue>238</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="readOnly" >
      <bool>false</bool>
     </property>
     <property name="decimals" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <double>10.000000000000000</double>
     </property>
     <property name="minimum" >
      <double>0.100000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.100000000000000</double>
     </property>
     <property name="value" >
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>individualButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>layerLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel" >
     <x>80</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>individualButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>neuronLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel" >
     <x>80</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>globalButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>layerLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel" >
     <x>80</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>globalButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>neuronLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel" >
     <x>80</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>individualButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectedLayerCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>83</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel" >
     <x>239</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>individualButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectedNeuronCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>83</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel" >
     <x>239</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>globalButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectedLayerCombo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>83</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel" >
     <x>239</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>globalButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectedNeuronCombo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>83</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel" >
     <x>239</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perLayerButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>layerLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel" >
     <x>80</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perLayerButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>neuronLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel" >
     <x>80</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perLayerButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectedLayerCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel" >
     <x>230</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perLayerButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectedNeuronCombo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel" >
     <x>230</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
